# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1467322558 14400
#      Thu Jun 30 17:35:58 2016 -0400
# Node ID 22a272df4ac44fcc28257065f8c2608a7590f590
# Parent  c33e721c01d79763320fad073b4452d5c8da691b
restore API macros to oct-conf-post.h removed in cset 6bce4d23af6b

* oct-conf-post.in.h (OCTAVE_USE_WINDOWS_API, OCTAVE_USE_OS_X_API):
Restore definitions.

diff --git a/oct-conf-post.in.h b/oct-conf-post.in.h
--- a/oct-conf-post.in.h
+++ b/oct-conf-post.in.h
@@ -75,17 +75,26 @@ typedef unsigned long ino_t;
    /* missing parameters in macros */
 #  pragma warning (disable: 4003)
    /* missing implementations in template instantiation */
 #  pragma warning (disable: 4996)
    /* deprecated function names (FIXME: ???) */
 #  pragma warning (disable: 4661)
 #endif
 
+#if defined (__APPLE__) && defined (__MACH__)
+#  define OCTAVE_USE_OS_X_API 1
+#endif
+
+/* Define to 1 if we expect to have <windows.h>, Sleep, etc. */
 #if defined (__WIN32__) && ! defined (__CYGWIN__)
+#  define OCTAVE_USE_WINDOWS_API 1
+#endif
+
+#if defined (OCTAVE_USE_WINDOWS_API)
 #  define OCTAVE_HAVE_WINDOWS_FILESYSTEM 1
 #elif defined (__CYGWIN__)
 #  define OCTAVE_HAVE_WINDOWS_FILESYSTEM 1
 #  define OCTAVE_HAVE_POSIX_FILESYSTEM 1
 #else
 #  define OCTAVE_HAVE_POSIX_FILESYSTEM 1
 #endif
 
